### PR TITLE
add searchForm input parameter for fetching characterCards

### PR DESCRIPTION
**Pull request by Ozanyurdakul98**

### CODE REVIEW

File: components/search-bar/search-bar.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  form.addEventListener('submit', event => {
+    event.preventDefault();
+    cardContainer.innerHTML = '';
+
+    const formData = new FormData(form);
+    const values = Object.fromEntries(formData);
+    let searchQuery = values.query;
+
+    console.log(searchQuery);
```
Comment (philmetscher) [inline components/search-bar/search-bar.js]: please remove console.log

File: components/search-bar/search-bar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    const fetchURL = `https://rickandmortyapi.com/api/character/?&name=${searchQuery}`;
+    console.log(fetchURL);
```
Comment (philmetscher) [inline components/search-bar/search-bar.js]: please remove console.log

File: components/search-bar/search-bar.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    fetchCharacters();
```
Comment (AHen-C0de) [inline components/search-bar/search-bar.js]: this function was renamed and a parameter was included, like so:
fetchCharactersAndRender(fetchURL)

Please change accordingly. :)

**Review by AHen-C0de — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (philmetscher) [inline .vscode/settings.json:1]: please remove this file in .gitignore and remove it from the repo

File: components/search-bar/search-bar.js (hunk 3 above)
Comment (philmetscher) [reply components/search-bar/search-bar.js]: pleas implement after @AHen-C0de has implemented the code for that

**Pull request merged**